modules/alphamat/tutorials/alphamat_tutorial.markdown

```diff
@@ -0,0 +1,53 @@
+Information Flow Alpha Matting {#tutorial_alphamat}
+============================
+
+This project was part of Google Summer of Code 2019.
+
+*Student:* Muskaan Kularia
+
+*Mentor:* Sunita Nayak
+
+Alphamatting is the problem of extracting the foreground from an image. The extracted foreground can be used for further operations like changing the background in an image.
+
+Given an input image and its corresponding trimap, we try to extract the foreground from the background. Following is an example:
+
+Input Image: ![](images/net_input_image.jpg)
```

Comment:
> input_images/*.png 

PNG is not suitable for real-life color images (files are huge).
Please convert these images into JPEG format (and amend commit to remove `PNG` files from git history of this PR).

BTW, "trimap" files are suitable for PNG format.

----

Keep duplicate images in "samples" only. Remove duplicates from `tutorial` (`net_result.jpg` is not a duplicate).
Use relative paths from `modules/alphamat` instead to access them: `samples/input_images/...`
(GitHub would fail to properly show .md files)

Reply:
The results of alpha matting are usually better with a Png file than a Jpg file. We should give the user a chance to see the better results. Pngs are also used in alpahmatting.com for evaluation. I had initially committed the resolution used at alphamatting.com. But I can resize them down if size is issue. 

Would two resized example png files each around 380KB be okay? Jpg images with similar resolution generating "okay" results are also 274KB and 236KB. 

Or if it is okay to leave only one image at alphamatting.com resolution(~700KB), that should be fine too.

Please let me know.

Reply:
I have removed the png images and included only one jpg image under samples..The earlier commits also have been squashed. So no longer getting the patch size warning.